.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
.vs files to ignore please?

Reply:
# Dependency directories
node_modules/
jspm_packages/
.vs/ 

ok?